### ISSUE TITLE
Add ability to list dependencies (like npm ls)

### ISSUE THREAD

**Issue by merlinpatt** [meteor/meteor-feature-requests#143]
I asked about this [on the forums](https://forums.meteor.com/t/how-can-i-find-package-dependencies/35148) but never got an answer. I can't find any easy way to list what packages depend on what. npm has this with [npm ls](https://docs.npmjs.com/cli/ls), and a similar solution could be done here

My solution, and I'm not even sure if it gets everything, is this

```
grep -r 'Package.blaze' .meteor/local/build/programs/web.browser/packages
```

If this is the best way, it would be nice if this were wrapped up in the tool with something like

```
meteor ls blaze
```

Or it could be more verbose and be `list-dependencies`

**Comment by hwillson** [comment]
Related to https://github.com/meteor/meteor/issues/2853 and https://github.com/meteor/meteor/issues/4387 (just linking these up).

**Comment by klaussner** [comment]
This feature is now available in Meteor 1.5.2! 🎉  Run `meteor list --tree` to get the dependencies of all packages.
